scripts/circle-ci-tests.sh

```diff
@@ -37,11 +37,11 @@ if [ "$CIRCLE_NODE_TOTAL" == "1" ] ; then
     echo "via the CircleCI UI and adjust scripts/circle-ci-tests.sh to match."
 
     echo "Running tests for common/lib/ and pavelib/"
-    paver test_lib --with-flaky --cov-args="-p" --with-xunitmp || EXIT=1
+    paver test_lib --with-flaky --cov-args="-p" --with-xunit || EXIT=1
```

Comment:
@benpatterson or @cpennington what's your recommendation here? We should co-ordinate these changes with #13898 so that no matter what the CI the tests are being run in the same way, otherwise there will be a higher risk of confusion when troubleshooting errors.